### PR TITLE
add formatting space for more transmitter vertices 

### PR DESCRIPTION
**Pull request by jen-holt-kobold**

### CODE REVIEW

**Comment by devjit-kobold** [comment]
Can you elaborate a bit more on what's happening here? My fortran is pretty weak...

**Comment by jen-holt-kobold** [comment]
> Can you elaborate a bit more on what's happening here? My fortran is pretty weak...

It was only allowing enough space for a two digit number of transmitter locations. When I connected your survey information there were 1000+ loop vertices and this code broke because it read in the number of vertices as ****. so I added two more spaces to allow for 10^3 loop vertices. I tested that it works, at least for that case. Fortran is so fiddly with stuff like this.

File: Leroi/Leroi.f90 [hunk 1]
```diff
@@ -2768,7 +2768,7 @@ SUBROUTINE READ_SYSTEM_AND_SURVEY_DATA
  9 FORMAT(/T12,'Frequency      Transmitter current in amps', &
           /T12,'---------      ---------------------------'/)
  10 FORMAT(//T3,'SURVEY_TYPE =',I2)
- 11 FORMAT(/T3,'NLINES =',I3,';   MRXL =',I4,';   NTX =',I3,';   SOURCE_TYPE =',I2,';   MXVRTX =',I2,';   NTRN =',I3)
+ 11 FORMAT(/T3,'NLINES =',I3,';   MRXL =',I4,';   NTX =',I3,';   SOURCE_TYPE =',I2,';   MXVRTX =',I4,';   NTRN =',I3)
```
Comment (jen-holt-kobold) [inline Leroi/Leroi.f90:2771]: I4 is allotting an integer with 4 spaces

Comment (devjit-kobold) [reply Leroi/Leroi.f90:2771]: do the integers need to be 0 or space padded (e.g. `0014` or ` 14`)

**Review by devjit-kobold — APPROVED** [review]

**Pull request merged**